### PR TITLE
[BrowserKit][HttpClient] Add examples to set cookies in the request

### PR DESCRIPTION
**Pull request by alexandre-daubois**
Fixes https://github.com/symfony/symfony-docs/issues/17875

### CODE REVIEW

File: http_client.rst [hunk 1]
```diff
@@ -686,6 +686,24 @@ You can either handle cookies yourself using the ``Cookie`` HTTP header or use
...
+    $client = HttpClient::create([
+        'headers' => [
+            'Cookie' => new Cookie('flavor', 'chocolate', strtotime('+1 day')),
+        ],
+    ]);
+
+.. note::
+
+    The :class:`Symfony\\Component\\HttpFoundation\\Cookie` class is a helper to
+    help you create cookies. You're also able to pass a simple string as the
```
Comment (OskarStark) [inline http_client.rst]: instead of the note, lets do something like this instead? (see above)

File: http_client.rst [hunk 2]
```diff
@@ -686,6 +686,24 @@ You can either handle cookies yourself using the ``Cookie`` HTTP header or use
...
 
+However, you're able to send cookies in your request. This is how you can do
+so using :class:`Symfony\\Contracts\\HttpClient\\HttpClient`::
+
+    use Symfony\Component\HttpClient\HttpClient;
+    use Symfony\Component\HttpFoundation\Cookie;
+
+    $client = HttpClient::create([
+        'headers' => [
+            'Cookie' => new Cookie('flavor', 'chocolate', strtotime('+1 day')),
```
Comment (OskarStark) [inline http_client.rst:697]:
```suggestion
            'Cookie' => new Cookie('flavor', 'chocolate', strtotime('+1 day')),
            
            // you're also able to pass a string instead
            'Cookie' => USE_A_STRING_HERE
```

Comment (alexandre-daubois) [reply http_client.rst:697]: Suits me, that's updated! :slightly_smiling_face:

**Comment by javiereguiluz** [comment]
Nice contribution as always. Thanks Alex!

**Pull request merged**